static/js/i18n/strings/en/units.json

```diff
@@ -34,5 +34,11 @@
   },
   "celsius-display": {
     "defaultMessage": "°c"
```

Comment:
sorry just noticed this. should be °C (capital C)

Reply:
Good catch!